telegram/ext/filters.py

```diff
@@ -387,7 +387,7 @@ def _merge(base_output: Union[bool, Dict], comp_output: Union[bool, Dict]) -> Fi
                 if isinstance(base[k], list):
                     base[k] += comp_value
                 else:
-                    base[k] = [base[k]] + comp_value
+                    base[k] = [base[k], *comp_value]
```

Comment:
this line is unchecked. Surely not a result of this PR, but it would be cool if we could increase the coverage :)